packages/runtime/package.json

```diff
@@ -153,6 +158,7 @@
     "path-to-regexp": "^6.2.1",
     "polished": "3.0.3",
     "react-player": "^1.12.0",
+    "react-reconciler": "0.31.0-rc-02c0e824-20241028",
```

Comment:
Use `react-reconciler` for the `findDOMNode` implementation for React 19+, see PoC here: https://github.com/makeswift/find-dom-node-spike/pull/1

Reply:
After what Seb mentioned, I tried looking into it and you should be able to use `ReactDOM['__DOM_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE'].findDOMNode`.

This quick experiment worked for me:

```js
'use client'

import React, { Component } from 'react'
import ReactDOM from 'react-dom'

export default function Page() {
  return <Div>Hi!</Div>
}

class Div extends Component {
  componentDidMount() {
    const findDOMNode =
      ReactDOM['__DOM_INTERNALS_DO_NOT_USE_OR_WARN_USERS_THEY_CANNOT_UPGRADE'].findDOMNode

    console.log(findDOMNode(this))
  }

  render() {
    return <div>{this.props.children}</div>
  }
}
```